### PR TITLE
Update documentation

### PR DESCRIPTION
**Pull request by ggordonhall**
Update README files with instructions on how to build and run bloop with natural language question answering. This is only a start; we'll be fleshing out the documentation in future updates.

### CODE REVIEW

File: apps/desktop/src-tauri/config.json [hunk 1]
```diff
@@ -1,4 +1 @@
-{
-    "skip-indexing": true,
-    "github_client_id": "5620006101aa9914ab43"
-}
+{}
```
Comment (nerdypepper) [inline apps/desktop/src-tauri/config.json:1]: can we make this config file optional?

File: server/README.md [hunk 2]
```diff
@@ -24,25 +24,37 @@ Exuberant Ctags 5.8, Copyright (C) 1996-2009 Darren Hiebert
...
+
+```
+docker run -p 6333:6333 -p 6334:6334 \
+    -e QDRANT__SERVICE__GRPC_PORT="6334" \
+    qdrant/qdrant
+```
+
+You can run `bleep` without linking it to a `Qdrant` instance but calls to the `/answer` endpoint will return an error.
+
+You'll also need to run a local instance of `answer_api` (`bleep` expects it on port 7879 by default) which handles requests to the OpenAI API.
```
Comment (nerdypepper) [inline server/README.md:38]: can we link to the answer_api repo here?

**Review by nerdypepper — APPROVED** [review]
lgtm, just one comment.

File: server/README.md (hunk 2 above)
Comment (ggordonhall) [reply server/README.md:38]: We need to open source first. Will do that and update after merging.

**Review by nerdypepper — APPROVED** [review]

File: apps/desktop/src-tauri/config.json (hunk 1 above)
Comment (ggordonhall) [reply apps/desktop/src-tauri/config.json:1]: It's not straightforward to optionally bundle additional files in Tauri (https://tauri.app/v1/guides/building/resources). Will stick this on the backburner.

**Pull request merged**